.github/labels.yml

```diff
@@ -202,24 +203,12 @@
 ###
 ### Hints
 ###
-- name: hint/good-first-issue
-  color: 0623cc
-  description: "Hint: Good First Issue"
```

Comment:
Don't we want to keep this one around? Could be nice if we wanted to pull in someone from Launchpad, for example, to work on the project.

Reply:
There is a "github magic label" at the top, I'd like to remove the duplicate label and use the one support by github.
https://github.com/testground/testground/blob/4dc2e51cf17f39c18bd149cfe00fc8ca83358235/.github/labels.yml#L11-L16